### DISCUSSION TITLE
SQL Lab Version 1.0

### DISCUSSION THREAD

**Discussion by athat90s** [apache/superset#18541] [Ideas]
Like other SQL tools

 - it would be great if query results could return as grid view and you could easily copy and paste results into other tools and retain the column headers when copying/pasting the results

 - Run multiple queries at the same time in SQL lab and and have each result return in multiple windows
 - within each query - save the schema and tables used in left pane (prefer a tree like SSIS etc..)
 - improved inteli sense when writing queries, highlight errors in color etc...
 -  - redesign query pane to reduce white space as challenging to use compared to other tools
thanks

**Comment by amitmiran137** [comment]
Not sure what is the agenda of that issue .

**Comment by MBRGA** [comment]
I second the part about making the results grid interactive. Being able to drag to select individual cells/rows/columns and pressing Ctrl-C to copy to clipboard (some software uses Ctrl+Insert is used to include headers) is a standard database query tool feature and would be so much more efficient than having to copy the whole results set to the clipboard.
